qiskit_addon_obp/utils/__init__.py

```diff
@@ -10,16 +10,4 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Utility functionality for conducting operator backpropagation.
-
-.. currentmodule:: qiskit_addon_obp.utils
-
-.. autosummary::
-   :toctree:
-
-   metadata
-   operations
-   simplify
-   truncating
-   visualization
-"""
+"""Utility functionality for conducting operator backpropagation."""
```

Comment:
Missing reminder comment:
```suggestion
# Reminder: update the RST file in docs/apidocs when adding new interfaces.
"""Utility functionality for conducting operator backpropagation."""
```

Reply:
I intentionally left it off because this module does not have an RST file. I'll write a dedicated comment explaining that.

Reply:
Ah I had not noticed that :+1: